edit_form.php

```diff
@@ -103,11 +103,6 @@ function definition() {
         // Add header text
         $mform->addElement('header','general', get_string('pluginname_short', 'enrol_ucsfsis'));
 
-        // Display notice if this enrolment is converted from CLEAE
```

Comment:
These lines should stay, so as the corresponding lang strings.  This is to let the user know this course enrollment was converted from CLEAE and has not been changed since.